Source/Clients/AspNetCore/Transactions/UnitOfWorkMiddleware.cs

```diff
@@ -21,6 +21,7 @@ public class UnitOfWorkMiddleware(IUnitOfWorkManager unitOfWorkManager, RequestD
     /// <returns>Awaitable task.</returns>
     public async Task InvokeAsync(HttpContext context)
     {
+        // TODO: Create Issue: Maybe there should be a try catch here and dispose the unit of work if an exception is thrown.
```

Comment:
What could cause an exception here? Are you seeing exceptions?
From what I can see from the current implementation all this does is create a new UnitOfWork instance and return this.